releases/v2024-3-3.md

```
@@ -0,0 +1,11 @@
+# 🚀 v2024.3.3 🚀
+
+## Changelog:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)